libmscore/figuredbass.cpp

```diff
@@ -1749,7 +1749,8 @@ void FiguredBass::writeMusicXML(XmlWriter& xml, bool isOriginalFigure, int crEnd
 FiguredBass* Score::addFiguredBass()
       {
       Element* el = selection().element();
-      if (el == 0 || (el->type() != ElementType::NOTE && el->type() != ElementType::FIGURED_BASS)) {
+      if (el == 0 || (el->type() != ElementType::NOTE && el->type() != ElementType::REST
+            && el->type() != ElementType::FIGURED_BASS)) {
```

Comment:
Three-space indent for multi-line conditionals.

And I think these should be changed to `isNote()`, `isRest()`, `isFiguredBass()` (if existing).

Reply:
`isFiguredBass()` does exist